.drone.yml

```diff
@@ -213,7 +213,7 @@ pipeline:
       event: [push, tag]
       status: [success, failure]
 
-  publish-gcs-builds-on-pass:
+  publish-gcs-builds-on-master-push-pass:
```

Comment:
I know it's just going to end up requiring more changes, but I'm inclined towards having master have a folder as well so structure is consistent. Do you have a feel for much of a cascade change this would be?

Reply:
Given the timelines, if you think this will rathole, let me know and we can do it in stages. My inclination in that case would be to publish master in the block below _as well_, so that we have it in both the old location and the new and can try moving pieces over without a comprehensive change.

Reply:
Changing where we publish artifacts from master would impact `vic-product`'s build scripts. I like the idea of publishing in both locations to allow for a gradual transition.